modules/oe_subscriptions_anonymous/src/EventSubscriber/EntityTypeSubscriber.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_subscriptions_anonymous\EventSubscriber;
+
+use Drupal\Core\Config\ConfigCrudEvent;
+use Drupal\Core\Config\ConfigEvents;
+use Drupal\extra_field\Plugin\ExtraFieldDisplayManager;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+/**
+ * Event subscriber for any config change. Maybe is too much.
+ */
+class EntityTypeSubscriber implements EventSubscriberInterface {
```

Comment:
There's this class that has dedicated events \Drupal\Core\Entity\EntityTypeEventSubscriberTrait.
You also need to subscribe to flag entities update/creation/deletion.
We need to be very precise or we will invalidate the cache too much.